app/operations/story_operations/read_all.rb

```diff
@@ -0,0 +1,46 @@
+module StoryOperations
+  class ReadAll
+    include Operation
+
+    delegate :past_iterations, :current_iteration_start, to: :iterations
+
+    def initialize(project:)
+      @project = project
+    end
+
+    def call
+      yield active_stories
+      yield project_iterations
+
+      Success(
+        active_stories: @active_stories,
+        past_iterations: past_iterations
+      )
+    end
+
+    private
+
+    attr_reader :project
+
+    def iterations
+      @project_iterations ||= Iterations::ProjectIterations.new(project: project)
+    end
+
+    def project_iterations
+      @project_iterations ||= iterations
+      Success(@project_iterations)
```

Comment:
Since the `iterations` method already does this, you don't need to do it again here, you can use the method directly:

```suggestion
      Success(iterations)
```

Reply:
makes sense